prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -1101,7 +1127,12 @@ class Hl7Serializer(
         terser.set(formPathSpec("OBX-24-2", aoeRep), report.getStringByHl7Field(row, "OBX-24-2"))
         terser.set(formPathSpec("OBX-24-3", aoeRep), report.getStringByHl7Field(row, "OBX-24-3"))
         terser.set(formPathSpec("OBX-24-4", aoeRep), report.getStringByHl7Field(row, "OBX-24-4"))
-        terser.set(formPathSpec("OBX-24-5", aoeRep), report.getStringByHl7Field(row, "OBX-24-5"))
+        val withLeadingZero = if (report.getStringByHl7Field(row, "OBX-24-5")?.length!! < 5) {
```

Comment:
Using the `!!` operator here means you're sure that `OBX-24-5` is never null. Can you make that guarantee? I would be worried that it could be and this would cause an entire report to fail.

Reply:
I'll take that into account, thanks.